File: src/main/java/eu/karelhovorka/zpevnik/util/StringUtil.java
package eu.karelhovorka.zpevnik.util;

import static eu.karelhovorka.zpevnik.util.Preconditions.checkNotNull;

public class StringUtil {

    public static String removeTrailingWhitespace(String text) {
        checkNotNull(text, "text");
        return text.replaceAll("\\s+$", "");
    }

}

File: src/main/java/eu/karelhovorka/zpevnik/util/StringUtil.kt
package eu.karelhovorka.zpevnik.util

import eu.karelhovorka.zpevnik.util.Preconditions.checkNotNull

object StringUtil {

    fun removeTrailingWhitespace(text: String): String {
        checkNotNull(text, "text")
        return text.replace("\\s+$".toRegex(), "")
    }

}
